samples/githubDispatch/command.js

```diff
@@ -0,0 +1,62 @@
+const { Octokit } = require("octokit");
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot trigger-fusebit-pipeline ...`.
```

Comment:
I think you mean `/fusebot github-dispatch` given the instructions in README.md

Reply:
Done